### DISCUSSION TITLE
Compilation error - Different version of the compiler Release Only

### DISCUSSION THREAD

**Discussion by AnorBanantje** [winsiderss/systeminformer#1335] [Questions / Answers] [answered]
Hey, i got compilation error.
1. Installed Windows SDK
2. Installed Visual Studio 2022
3. Executed build_release.cmd

![image](https://user-images.githubusercontent.com/109865290/184332965-0dca6d06-c80e-4a02-8064-6f07ff6f761c.png)

Anyone had this problem?

ps. i can successfully compile it as debug in Visual Studio

**Comment by dmex** [comment, marked as answer]
`\build\build_thirdparty.cmd`

**Reply by AnorBanantje** [reply]
Thanks it's works!
